src/components/Marker.tsx

```diff
@@ -12,7 +12,7 @@ export default function Marker({
   accessibilityLabel = null,
   subtitleVisibility = FeatureVisibility.Adaptive,
   titleVisibility = FeatureVisibility.Adaptive,
-
+  clusteringIdentifier,
```

Comment:
Could you please set the default value to `null` to match what MapKit JS does?

```suggestion
  clusteringIdentifier = null,
```